idp-service/src/main/resources/migrations/scorecard/checks.json

```
@@ -197,9 +197,89 @@
         "value": "true"
       }
     ],
-    "expression": "harness.noTestsFailingInCiPipeline == 0",
+    "expression": "harness.noTestsFailingInCiPipeline == true",
     "tags": ["Software quality", "Development maturity"],
     "defaultBehaviour": "FAIL",
     "failMessage": "Tests are failing on CI Pipeline"
+  },
+  {
+    "id": "65009e1bc28a1a3f5607f80c",
+    "accountIdentifier": "__GLOBAL_ACCOUNT_ID__",
+    "identifier": "__HARNESS__712025a1-c192-4212-8c29-252a37a44225",
+    "name": "Is on call set",
+    "description": "checks if on call is set on PagerDuty",
```

Let's write full sentences in the description.

```suggestion
    "description": "Checks if on-call is set on PagerDuty for the service.",
```